src/test/java/org/assertj/core/internal/doubles/Doubles_assertIsCloseTo_Test.java

```diff
@@ -73,8 +73,36 @@ void should_pass_if_actual_and_expected_are_NaN() {
     doubles.assertIsCloseTo(someInfo(), NaN, NaN, byLessThan(ONE));
   }
 
+  @Test
+  void should_pass_if_actual_and_expected_are_equal_within_range1(){
+    doubles.assertIsCloseTo(someInfo(), 1.1d, 1.0d, within(0.1d));
+    doubles.assertIsCloseTo(someInfo(), 1.0d, 1.1d, byLessThan(0.1000001d));
+  }
+
+  @Test
+  void should_pass_if_actual_and_expected_are_equal_within_range2(){
+    doubles.assertIsCloseTo(someInfo(), 0.375d, 0.125, within(0.25));
+    doubles.assertIsCloseTo(someInfo(), 0.125d, 0.375, byLessThan(0.2500001));
+  }
```

Comment:
these tests assertions belong either in `should_pass_if_difference_is_less_than_given_offset` or `should_pass_if_difference_is_equal_to_given_offset`, move the assertions in the proper test methods and remove these tests.